Makefile

```diff
@@ -1,5 +1,5 @@
 export GO111MODULE=on
-export GOPROXY=https://proxy.golang.org
+export GOPROXY="${CUSTOM_GOPROXY:-https://proxy.golang.org}"
```

Comment:
Can you test if
 
```
GOPROXY ?= https://proxy.golang.org
export GOPROXY=$(GOPROXY)
```
would work, then you wouldn't need to set a `CUSTOM_GOPROXY` but instead use the one you desire

Reply:
@Jguer Thanks for the hint. I just updated the code to remove unnecessary `CUSTOM_GOPROXY`.